x-pack/solutions/security/plugins/security_solution/public/sourcerer/containers/index.tsx

```diff
@@ -64,14 +66,23 @@ export const useSourcererDataView = (
   );
 
   useEffect(() => {
+    if (newDataViewPickerEnabled) {
+      return;
+    }
     if (selectedDataView == null || missingPatterns.length > 0) {
```

Comment:
@lgestc need to investigate whether one of my changes introduced this, but since `defaultDataView` is never set in sourcerer...opening up a timeline causes this to run in an infinite loop. Still need to dig into what the code path for this is, but it most likely involves `useCreateTimeline` and/or `useUpdateTimeline`